packages/rancher-gatekeeper/generated-changes/patch/Chart.yaml.patch

```diff
@@ -5,11 +5,11 @@
 +  catalog.cattle.io/auto-install: rancher-gatekeeper-crd=match
 +  catalog.cattle.io/certified: rancher
 +  catalog.cattle.io/display-name: OPA Gatekeeper
-+  catalog.cattle.io/kube-version: '>= 1.16.0-0'
++  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.22.0-0'
```

Comment:
let's just mark this chart doNotRelease in release-v2.6 and not modify the `packages/` changes. Same for istio.